door-open-service/node-backend/src/slackInterface.js

```diff
@@ -23,6 +23,19 @@ export default function (config, slackBot, sockServer) {
           "Consider dispatching a drone to pick up a human.");
 
     } else {
+
+      config.doorTimes = { "garage": 25 * 1000 };
```

Comment:
Since this is currently implemented in `slackInterface`, I'd put the new settings inside `config.slackBot` - after all, this is only supported on Slack currently. Once we extract this logic to be used in other interfaces we can also move the respective config to another key.